src/main/java/org/sagebionetworks/bridge/sdk/models/UploadRequest.java

```diff
@@ -89,4 +77,94 @@ public String toString() {
         return String.format("UploadRequest [name=%s, contentMd5=%s, contentType=%s, contentLength=%s]", 
                 name, contentMd5, contentType, contentLength);
     }
+
+    /** Builder for UploadRequest. */
+    public static class Builder {
```

Comment:
Discussed over chat. We don't have a formal style guideline for this, but most of the builders we have (server and client) are static inner classes. We have a few exceptions where we have abstract builders as top-level classes and concrete implementations that are static inner classes.